test/Microsoft.Dnx.Testing.CommonUtils/DnxSdk/Dnx.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using Microsoft.Dnx.Runtime;
+
+namespace Microsoft.Dnx.Testing
+{
+    public class Dnx
+    {
+        private readonly string _sdkPath;
+
+        public Dnx(string sdkPath)
+        {
+            _sdkPath = sdkPath;
+        }
+
+        public ExecResult Execute(string commandLine, bool dnxTraceOn = true)
+        {
+            string command;
+            if (RuntimeEnvironmentHelper.IsWindows)
+            {
+                command = "cmd";
+                commandLine = $"/C {Path.Combine(_sdkPath, "bin", "dnx.exe")} {commandLine}";
+            }
+            else
+            {
+                command = "bash";
```

Comment:
I see. Just to make sure, dnx is an executable whereas dnu is a script so this fix should only be made here. Also, should IsMono be used instead of IsWindows as was the case for grabbing runtime sdks? Isn't mono unsupported on Windows so why did we bother with that?